package.json

```diff
@@ -75,21 +75,24 @@
     "mv": "^2.1.1",
     "ncp": "^2.0.0",
     "nock": "^13.0.0",
+    "node-polyfill-webpack-plugin": "^1.1.0",
```

Comment:
This doesn't look to be used anywhere - is it just ambiently available to webpack?

Reply:
I thought it was the solution to `Buffer is not defined`, but it's not so I removed it!